Core/Services/Auth/RegisterService.cs

```diff
@@ -13,15 +13,13 @@
 
 namespace Core.Services.Auth
 {
-    class RegisterService : IRegisterService
+    class RegisterService : AuthServicesProvider, IRegisterService
     {
-        private readonly UserManager<User> _userManager;
-        private readonly IConfiguration _config;
-        private readonly IEmailService _emailService;
-        public RegisterService(UserManager<User> userManager, IConfiguration config, IEmailService emailService){
-            _userManager = userManager;
-            _config = config;
-            _emailService = emailService;
+        private readonly IAdditionalAuthMetods _additionalAuthMetods;
+        public RegisterService(UserManager<User> userManager, IConfiguration config, IEmailService emailService, IAdditionalAuthMetods add)
+            :base(userManager, config, emailService)
+        {
+            _additionalAuthMetods = add;
```

Comment:
Same as before, change 'add' to proper name

Reply:
Done